### PR TITLE
[CN-799]: Add doc for jet engine configuration

### PR DESCRIPTION
**Pull request by kutluhanmetin**

### CODE REVIEW

**Review by cagric0 — APPROVED** [review]

File: docs/modules/ROOT/pages/jet-engine-configuration.adoc [hunk 1]
```diff
@@ -0,0 +1,74 @@
+= Configuring the Jet Engine
+:description: This section describes Hazelcast’s Jet engine specific configuration using Hazelcast Platform Operator.
+
+{description}
+
+== Enabling Jet Engine
```
Comment (cagric0) [inline docs/modules/ROOT/pages/jet-engine-configuration.adoc:6]:
```suggestion
== Configuring the Jet Engine
```
I would use `Configuring` to be consistent with other pages

File: docs/modules/ROOT/pages/jet-engine-configuration.adoc [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+When you run Hazelcast using Platform Operator, Jet Engine is enabled by default.
+
+== Configuration Options
```
Comment (cagric0) [inline docs/modules/ROOT/pages/jet-engine-configuration.adoc:9]:
```suggestion
```
Same for here, I would remove this line to be consistent with other doc pages

**Review by cagric0 — CHANGES_REQUESTED** [review]
I left two suggestion. Other than that LGTM

**Review by cagric0 — APPROVED** [review]

File: docs/modules/ROOT/pages/jet-engine-configuration.adoc [hunk 3]
```diff
@@ -0,0 +1,72 @@
+= Configuring the Jet Engine
+:description: This section describes Hazelcast’s Jet engine specific configuration using Hazelcast Platform Operator.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/jet-engine-configuration.adoc:2]:
```suggestion
:description: You can configure Hazelcast’s Jet engine using the Hazelcast Platform Operator.
```

File: docs/modules/ROOT/pages/jet-engine-configuration.adoc [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+{description}
+
+== Configuring the Jet Engine
+When you run Hazelcast using Platform Operator, Jet Engine is enabled by default.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/jet-engine-configuration.adoc:7]:
```suggestion
The Jet engine is enabled by default and has the following configuration options.
```
As we're in the Operator docs, I don't think we need to repeat Hazelcast Platform Operator.

File: docs/modules/ROOT/pages/jet-engine-configuration.adoc [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+|===
+|Option|Default|Description
+
+|enabled
+|true
+|When false, disables Jet Engine.
+
+|resourceUploadEnabled
+|false
+|When true, enables resource uploading for Jet jobs.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/jet-engine-configuration.adoc:19]:
```suggestion
|Set to `true` to enable the uploading of resources for Jet jobs.
```

File: docs/modules/ROOT/pages/jet-engine-configuration.adoc [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+|
+|The number of threads Jet creates in its cooperative multithreading pool.
+
+|instance/flowControlPeriodMS
+|100
+|The duration of the interval between flow-control packets.
+
+|instance/backupCount
+|1
+|The number of synchronous backups to configure on the IMap that Jet needs internally to store job metadata and snapshots.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/jet-engine-configuration.adoc:31]:
```suggestion
|The number of synchronous backups to configure on the IMap, which Jet needs internally to store job metadata and snapshots.
```

File: docs/modules/ROOT/pages/jet-engine-configuration.adoc [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+|instance/scaleUpDelayMS
+|10000
+|The delay after which the auto-scaled jobs restart if a new member joins the cluster.
+
+|instance/losslessRestartEnabled
+|false
+|Specifies whether the Lossless Cluster Restart feature is enabled.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/jet-engine-configuration.adoc:39]:
```suggestion
|Whether the Lossless Cluster Restart feature is enabled.
```

File: docs/modules/ROOT/pages/jet-engine-configuration.adoc [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+|instance/maxProcessorAccumulatedRecords
+|
+|Specifies the maximum number of records that can be accumulated by any single processor instance.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/jet-engine-configuration.adoc:43]:
```suggestion
|The maximum number of records that can be accumulated by any single processor instance.
```

**Review by angelasimms — CHANGES_REQUESTED** [review]
Just a few suggestions.

**Review by angelasimms — APPROVED** [review]
LGTM

**Pull request merged**